Activities/Database/ConnectionDialog/UiPath.Data.ConnectionUI.Dialog/AdoDotNetConnectionProperties.cs

```diff
@@ -179,6 +186,10 @@ public virtual void Test()
             {
                 connection = new OracleConnection(testString);
             }
+            else if (_providerName.Equals("Microsoft.Data.SqlClient"))
```

Comment:
Maybe a switch instead of an if here, since we already have 3 cases .

Reply:
Fixed